src/Illuminate/Database/Eloquent/Model.php

```diff
@@ -481,15 +481,18 @@ public function newInstance($attributes = array(), $exists = false)
 	 * Create a new model instance that is existing.
 	 *
 	 * @param  array  $attributes
+	 * @param  string  $connection
```

Comment:
> It's not like that throughout the codebase

Yeh, I know. There's no point wasting time enforcing it everywhere, as with the short array syntax. We're just making sure new code is correct. Changing all the existing code is a total waste of time.